src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```

Comment:
Hmm i dont think this component should sits under ActionsPage folder

Reply:
Why?

Reply:
Not important anymore, thought we will have separate page for Motions